run_uts.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -x
+set -e
+date
+pwd
+git status
+
+./create_binary.sh
+docker stop pyinstaller docopt
+docker rm pyinstaller docopt
+docker run -rm -v `pwd`/:/code calico-build bash -c '/tmp/etcd & nosetests -c nose.cfg'
+
+# BE CAREFUL
+rm -rf .coverage cover/ default.etcd/
```

Comment:
When I run this file locally, nose creates these dirs, I think because of what's in `nose.cfg`. They're useless in that case. I can't change `nose.cfg` because it's for circleci. I could make a `local_nose.cfg` but that seems excessive. I tried overriding it with arguments but didn't get it to work. And I'd really rather not type `rm -rf` manually a bunch of times.